test/class_test.ts

```diff
@@ -1952,4 +1952,20 @@ describe('classes', () => {
       })();
     `);
   });
+
+  it('produce a proper hack to trick Babel and into TypeScript that would be OK with UglifyJS -- and continues to produce results as would be expected', () => {
+      validateCS1(`
+      class A
+        constructor: ->
+          @value ?= 1
+          @value += 1
+      
+      class B extends A
+        constructor: ->
+          @value = 3
+          super
+      b = new B()
+      setResult(b.value)
+    `, 4, {options: {disableBabelConstructorWorkaround: false}, skipNodeCheck: true});
```

Comment:
`disableBabelConstructorWorkaround` is false by default, so it should be unnecessary here, and I think the test name makes it clear enough.